Sources/Bob/APIs/GitHub/GitHub.swift

```diff
@@ -334,8 +334,7 @@ public class GitHub {
         
         let author = try JSON(node: [
             "name": author.name,
-            "email": author.email,
-            "date": author.date
```

Comment:
I'm confused though. Because prior to my change, this date was set to `date`, where `date` was created inside the newCommit() method:

```
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)!
        let date = dateFormatter.string(from: Date()).appending("Z")
     ```

I just moved that code into the Author struct. So the date should be the time when the Author instance was created. But is there a single Author created at launch of Bob instance? Oh dang. I didn't think of that. :(

So we can put revert back to the code I changed?
```
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)!
        let date = dateFormatter.string(from: Date()).appending("Z")
        
        let author = try JSON(node: [
            "name": author.name,
            "email": author.email,
            "date": date
        ])
```